pkgs/servers/caddy/default.nix

```
@@ -35,7 +42,34 @@ buildGoModule {
     "-X github.com/caddyserver/caddy/v2.CustomVersion=${version}"
   ];
 
-  nativeBuildInputs = [ installShellFiles ];
+  nativeBuildInputs = [ gnused installShellFiles ];
```

```suggestion
  nativeBuildInputs = [ installShellFiles ];
```
gnused is part of stdenv and does not need to be added

---

Wouldn't gnused be needed to ensure `-i` works on darwin and bsd? There are some flag incompatibilities: https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux

---

Yes, `sed -i` without an explicit extension (`-i.bak`) won't work on macOS. So we either have to keep `gnused`, or change the sed command on line 49.